lib/rules/new-ember-data-packages.js

```diff
@@ -0,0 +1,167 @@
+'use strict';
+
+const MAPPINGS = require('@ember-data/rfc395-data');
+const { buildFix, buildMessage } = require('../utils/new-module');
+
+/**
+ * This function returns an object like this:
+ * {
+ *   'ember-data/model': {
+ *     default: ['@ember-data/model'],
+ *     attr: ['@ember-data/model', 'attr'],
+ *   },
+ * }
+ */
+function oldDataImportsReducer(acc, mapping) {
+  let obj = Object.create(null);
+  obj[mapping.export] = [mapping.replacement.module, mapping.replacement.export];
+  acc[mapping.module] = obj; // eslint-disable-line no-param-reassign
+
+  return acc;
+}
+
+const OLD_DATA_IMPORTS = MAPPINGS.reduce(oldDataImportsReducer, Object.create(null));
+
+/**
+ * This function returns an object like this:
+ * {
+ *   "DS.Model": {
+ *     global: "DS.Model",
+ *     export: "default",
+ *     localName: "Model",
+ *     module: "@ember-data/model" <- replacement
+ *   }
+ * }
+ */
+function globalsReducer(acc, mapping) {
+  const { global } = mapping;
+  if (global in acc) {
+    return acc;
+  }
+
+  let obj = {
+    global,
+    export: mapping.replacement.export,
+    localName: mapping.localName,
```

Comment:
https://github.com/ember-data/ember-data-rfc395-data/blob/master/mappings.json#L154

Does `attr` need a localeName of `attr`?

Reply:
Indeed! I've opened a PR to correct that: https://github.com/ember-data/ember-data-rfc395-data/pull/8